erln8.c

```diff
@@ -1444,8 +1444,7 @@ int main(int argc, char* argv[]) {
     if(!result) {
       g_hash_table_destroy(erlangs);
       g_hash_table_destroy(runtime_options);
-      g_free(s);
-      g_error("Can't run %s, check to see if the file exists\n", s);
+      g_error("Can't run %s, hello check to see if the file exists\n", s);
```

Comment:
The `hello` is some debugging I added and left in accidentally, I will remove it.

Isn't the `g_error` this function: https://developer.gnome.org/glib/stable/glib-Message-Logging.html#g-error ?